include/sempr/entity/spatial/filter/GeodeticFilter.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef SEMPR_ENTITY_SPATIAL_GEODETICFILTER_HPP_
+#define SEMPR_ENTITY_SPATIAL_GEODETICFILTER_HPP_
+
+//#include <ogr_spatialref.h>
+#include <geos/geom/CoordinateSequenceFilter.h>
+#include <geos/geom/Coordinate.h>
+
+// weeeeeell... it's not an entity. more like... util?
+namespace sempr {
+
+namespace geom = geos::geom;
+
+class GeodeticException : public std::exception
+{
+    private:
+    std::string message_;
+
+    public:
+    explicit GeodeticException(const std::string &m) : message_(m) {}
+    virtual const char *what() const throw()
+    {
+        return message_.c_str();
+    }
+};
+
+
+class EmptyFilter : public geom::CoordinateSequenceFilter 
```

Comment:
Why is there no `GeodeticFilter` in the `GeodeticFilter.hpp`?

Reply:
Solved
Discussion: The geom filter files are grouped and named by context not by classes (otherwise there would be many files. For each small filter class)